<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>app_pages</class>
 <widget class="QStackedWidget" name="app_pages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <widget class="QWidget" name="page_spam">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: rgb(37, 38, 52);
	padding: 8px;
	border: 2px solid #c3ccdf;
	border-radius:  10px;
	color: rgb(255, 255, 255)
}

QLabel{
	font: 700 14pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton{
	background-color: rgb(37, 38, 52);
	padding: 8px;
	border: 2px solid #c3ccdf;
	border-radius:  15px;
	color: rgb(255, 255, 255)
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_invite_channel">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="0" column="1" alignment="Qt::AlignRight|Qt::AlignTop">
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
       <item row="1" column="1" alignment="Qt::AlignRight|Qt::AlignTop">
        <widget class="QPushButton" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777214</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="locale">
          <locale language="Russian" country="Russia"/>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>700</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777214</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="locale">
          <locale language="Russian" country="Russia"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="autoScrollMargin">
          <number>18</number>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::Batched</enum>
         </property>
         <property name="batchSize">
          <number>103</number>
         </property>
         <property name="currentRow">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_invite_chat">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Page invite chat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_setup">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: rgb(37, 38, 52);
	padding: 8px;
	border: 2px solid #c3ccdf;
	border-radius:  10px;
	color: rgb(255, 255, 255)
}

QLabel{
	font: 700 14pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton{
	background-color: rgb(37, 38, 52);
	padding: 8px;
	border: 2px solid #c3ccdf;
	border-radius:  15px;
	color: rgb(255, 255, 255);
}

QPushButton: hover{
	background-color: rgb(255, 255, 127);
}

QPushButton: pressed{
	background-color: rgb(0, 0, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_id">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="id_ldt">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>Пример id: 5642667</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_hash">
         <property name="text">
          <string>Hash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hash_ldt">
         <property name="placeholderText">
          <string>Пример hash: jhgdsjdsgj436hgdsgl</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_num">
         <property name="text">
          <string>Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="num_ldt">
         <property name="placeholderText">
          <string>Пример номера телефона: +79090456</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
